packages-exp/auth-exp/test/integration/webdriver/persistence.test.ts

```diff
@@ -182,34 +182,8 @@ browserDescribe('WebDriver persistence test', (driver, browser) => {
       );
     });
 
-    it('migrate stored user to localStorage if indexedDB is readonly', async () => {
```

Comment:
Clarifying q (trying to understand more than anything!): Not catching why this test would be impacted by the fix to session persistence? This test concerns localStorage and indexDB only, right?

Reply:
This change also makes it so we only look at "available" persistence layers before initializing the SDK. If `indexedDB` is readonly, it would be filtered out early on, so this behavior (migrating from readonly indexedDB to localStorage) doesn't happen anymore.

Reply:
Ohh got it, thanks for the clarification!